brands/shop/erotic.json

```diff
@@ -9,6 +9,25 @@
       "shop": "erotic"
     }
   },
+  "shop/erotic|Amazing Intimate Essentials": {
```

Comment:
name here should match `name` tag below... and should be whatever these stores are signed as.

Reply:
Thanks @bhousel! I took care of that.